### DISCUSSION TITLE
orthogonal_bitmap_intersect异常

### DISCUSSION THREAD

**Discussion by SamuelBin** [apache/doris#12450] [A - General / Q&A]
select orthogonal_bitmap_intersect(user_id, tag_id, 1,2) from test.user_tag_bitmap;

Error Failed to execute sql: java.sql.SQLException: (conn=90) errCode = 2, detailMessage = No matching function with signature: orthogonal_bitmap_intersect(bitmap, bigint(20), tinyint(4), tinyint(4)).

**Comment by LemonLiTree** [comment]
向量化版本这三个函数有点问题，已经修复了，在马上发布的1.1.2可以用。如果你现在需要用的话可以关闭向量化。
set enable_vectorized_engine = false;
